game/world/objects/interactive.cpp

```diff
@@ -108,10 +120,12 @@ void Interactive::load(Serialize &fin) {
     invent.load(fin,*this,world); else
     invent.clear(world.script(),*this,true);
 
-  fin.setEntry("worlds/",fin.worldName(),"/mobsi/",vobObjectID,"/visual");
-  visual.load(fin,*this);
-  visual.setObjMatrix(transform());
-  visual.syncPhysics();
+  if (fin.version() < 39 && vobType != phoenix::vob_type::oCMobContainer) {
```

Comment:
what this condition is for?

Reply:
That's actually a good catch! It fixed a problem where chests wouldn't load in properly but I actually found the root cause so this is no longer needed. I've removed the check.